src/components/state-management/ServerState/ServerStateExample.tsx

```diff
@@ -0,0 +1,79 @@
+import { Card, Image, Inline, SearchField, Select, Stack, Text } from "@marigold/components";
+import { useQuery } from "@tanstack/react-query";
+import { useState } from "react";
+
+interface MovieI {
+    href: string;
+    year: number;
+    title: string;
+    category: string;
+    thumbnail: string;
+    thumbnail_width: number;
+    thumbnail_height: number;
+    extract: string;
+}
+function ServerStateExample() {
+
+    const [filters, setFilters] = useState<{ title: string, category: string }>({
+        title: "",
+        category: "",
+    });
+
+    const fetchData = async (url: string, queryParams: Record<string, string>) => {
+        const queryString = new URLSearchParams(queryParams).toString();
+        const apiURL = url + (queryString ? `?${queryString}` : '');
```

Comment:
If you wanna use a template string you could write it so:

```suggestion
        const apiURL = `${url}${(queryString ? `?${queryString}` : '')}`;
```